<ui version="4.0" >
 <class>settings</class>
 <widget class="QWidget" name="settings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Tube</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Inner Radius:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2" >
       <widget class="QPushButton" name="insertButton" >
        <property name="text" >
         <string>Insert Tube</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Corners:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QSpinBox" name="corners" >
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="minimum" >
         <number>3</number>
        </property>
        <property name="value" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Outer Radius:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="oRadius" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>1000.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>0.100000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QDoubleSpinBox" name="height" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>1000.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="iRadius" >
        <property name="decimals" >
         <number>4</number>
        </property>
        <property name="maximum" >
         <double>1000.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2" >
       <widget class="QCheckBox" name="sharedVertices" >
        <property name="text" >
         <string>Shared Vertices</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Parts:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QSpinBox" name="parts" >
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>oRadius</tabstop>
  <tabstop>iRadius</tabstop>
  <tabstop>height</tabstop>
  <tabstop>corners</tabstop>
  <tabstop>parts</tabstop>
  <tabstop>sharedVertices</tabstop>
  <tabstop>insertButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
